tests/support/bower.json

```diff
@@ -5,8 +5,9 @@
     "purescript-console": "1.0.0-rc.1",
     "purescript-eff": "1.0.0-rc.1",
     "purescript-functions": "1.0.0-rc.1",
-    "purescript-prelude": "1.0.0-rc.3",
+    "purescript-prelude": "1.1.0",
     "purescript-st": "1.0.0-rc.1",
-    "purescript-partial": "1.1.2"
+    "purescript-partial": "1.1.2",
+    "purescript-newtype": "0.1.0"
```

Comment:
Yeah, I just made a release so we could add the test thing - we'll need to make a 2.0 prelude anyway, so when that is released the dependency will need bumping in `newtype`, at which point it'll be worth releasing as 1.0... phew :wink: